packages/quantic/cypress/e2e/default-2/standalone-search-box/standalone-search-box-actions.ts

```diff
@@ -15,7 +15,7 @@ const standaloneSearchBoxActions = (selector: StandaloneSearchBoxSelector) => {
             updateText += letter;
             cy.wrap(searchbox)
               .invoke('val', updateText)
-              .trigger('keyup', {which: letter.charCodeAt(0)});
+              .trigger('input', {which: letter.charCodeAt(0)});
```

Comment:
wut?

Reply:
the input component no longer listens to keyup events, that's part of the simplification I added.